### DISCUSSION TITLE
auth credentials are changing everytime when trying to  deploy the flask application using sqlalchemy

### DISCUSSION THREAD

**Discussion by sumit077777** [planetscale/discussion#452] [General]
r sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (1105, 'unauthenticated: invalid auth credentials')

**Comment by mscoutermarsh** [comment]
Hi @sumit077777 

Could you share more about how you are deploying your app? How are you setting the DB credentials?

We have nothing on our end that would expire credentials on a deploy.
